### PR TITLE
LPS-166151 Product QA | Automation Test Creation - LPS-165176 Inject Headless resources of Objects into templates

### PR DESCRIPTION
**Pull request by magjed4289**
https://issues.liferay.com/browse/LPS-166151

### CODE REVIEW

**Comment by magjed4289** [comment]
ci:test:headless-functional:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/HeadlessResourceTemplates.testcase [hunk 1]
```diff
@@ -24,11 +24,113 @@ definition {
 
 			ApplicationDisplayTemplates.openWidgetTemplatesAdmin(siteURLKey = "guest");
 
-			ApplicationDisplayTemplates.deleteCP(widgetTemplateName = "Blogs Test ADT");
+			while (IsElementNotPresent(locator1 = "Message#EMPTY_INFO", value1 = "There are no templates.")) {
```
Comment (yanan) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/HeadlessResourceTemplates.testcase:27]: Looks good!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/HeadlessResourceTemplates.testcase [hunk 2]
```diff
@@ -24,11 +24,113 @@ definition {
...
+			var objectDefinitionEntryId = ObjectDefinitionAPI.createObjectEntryWithName(
+				en_US_plural_label = "universities",
+				name = "Liferay University");
+		}
+
+		task ("And Given an information template with getObjectEntriesPage created") {
+			InformationTemplates.openInformationTemplatesAdmin(siteURLKey = "guest");
+
+			InformationTemplates.addCP(
+				adtFile = "adt_objects.ftl",
```
Comment (yanan) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/HeadlessResourceTemplates.testcase:77]: Seems the adtFile here can be removed, right?

**Review by Hijky — APPROVED** [review]

**Comment by magjed4289** [comment]
ci:test:sf

**Comment by magjed4289** [comment]
ci:test:headless-functional:nocompile

**Comment by magjed4289** [comment]
ci:close

**Comment by magjed4289** [comment]
ci:reopen

**Comment by magjed4289** [comment]
ci:test:stable

**Comment by magjed4289** [comment]
will send in another PR

**Pull request closed without merging**